services/brig/test/integration/API/Provider.hs

```diff
@@ -97,11 +97,12 @@ import Wire.API.Team.Feature (featureNameBS)
 import Wire.API.Team.Feature qualified as Public
 import Wire.API.Team.Permission
 import Wire.API.User as User hiding (EmailUpdate, PasswordChange, mkName)
+import Wire.API.User.Auth (CookieType (..))
```

Comment:
Not sure why this change.

Reply:
I added a test which uses the nginz endpoint instead of brig (trying to reproduce the failing test that QA reported). I thought it doesn't hurt to leave this test in here.